server/notification-providers/teams.js

```diff
@@ -224,7 +224,7 @@ class Teams extends NotificationProvider {
 
             const payload = this._notificationPayloadFactory({
                 heartbeatJSON: heartbeatJSON,
-                monitorName: monitorJSON.name,
+                monitorName: monitorJSON.pathName,
```

Comment:
no idea if this would break something

```suggestion
                monitorName: monitorJSON.name,
```